docs/src/bpmn-workflows/call-activities/call-activities.md

```diff
@@ -32,9 +32,9 @@ When the call activity is activated then **all variables** of the call activity
 
 Input mappings can be used to create new local variables in the scope of the call activity. These variables are also copied to the created workflow instance.
 
-By default, all variables of the created workflow instance are propagated to the call activity. This behavior can be customized by defining output mappings at the call activity. The output mappings are applied on completing the call activity.
-This is especially important in the case of a call activity in a parallel flow to avoid overriding variables (e.g. when it is marked as
-[parallel multi-instance](/bpmn-workflows/multi-instance/multi-instance.html#variable-mappings)).
+If the attribute `propagateAllChildVariables` is set (default: `true`) then all variables of the created workflow instance are propagated to the call activity. This behavior can be customized by defining output mappings at the call activity. The output mappings are applied on completing the call activity.
```

Comment:
> This behavior can be customized by defining output mappings at the call activity

Is that true? Is the behavior still working for variables that are not defined in the output mapping when there are other output mappings defined and this prop is set to true?

Reply:
If any output mapping is defined then the mappings are applied and only the variables are propagated that are defined in the mappings.

Do you have a suggestion on how to make this more clear in the docs?

Reply:
How about something like this: propagation of all variables is ignored when output mapping is defined, output mapping can be used to customize propagation, propagation is applied on completing.

```suggestion
If the attribute `propagateAllChildVariables` is set (default: `true`) then all variables of the created workflow instance are propagated to the call activity. This behavior is ignored when output mappings are defined at the call activity. Output mappings can thus be used to customize which variables are propagated. The variable propagation is applied on completing the call activity.
```